tests/TestHarness/TestHelper.py

```diff
@@ -37,87 +37,101 @@ class TestHelper(object):
     @staticmethod
     # pylint: disable=too-many-branches
     # pylint: disable=too-many-statements
-    def parse_args(includeArgs, applicationSpecificArgs=AppArgs()):
+    def createArgumentParser(includeArgs, applicationSpecificArgs=AppArgs()) -> argparse.ArgumentParser:
         """Accepts set of arguments, builds argument parser and returns parse_args() output."""
         assert(includeArgs)
         assert(isinstance(includeArgs, set))
         assert(isinstance(applicationSpecificArgs, AppArgs))
 
-        parser = argparse.ArgumentParser(add_help=False, formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        parser.add_argument('-?', action='help', default=argparse.SUPPRESS,
+        thParser = argparse.ArgumentParser(add_help=False, formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        thGrpTitle = "Test Helper Arguments"
+        thGrpDescription="Test Helper configuration items used to configure and spin up the regression test framework and blockchain environment."
+        thGrp = thParser.add_argument_group(title=thGrpTitle, description=thGrpDescription)
+        thGrp.add_argument('-?', action='help', default=argparse.SUPPRESS,
```

Comment:
Conscider adding the long option `--help`. Though not a posix standard, it is a [GNU standard](https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html) and I have come to expect it is available.  
```suggestion
        thGrp.add_argument('-?', '--help', action='help', default=argparse.SUPPRESS,
```

If you do make this change, you might consider adding `--version` as well, though I'm unsure it has much use.

Reply:
Addressed: 9063dd87e1b

Left of `--version` for now as I'm not sure how much use or merit it currently provides.